weighted_levenshtein/clev.pyx

```diff
@@ -132,12 +132,29 @@ cdef inline DTYPE_t row_insert_range_cost(
 
 # End Array2D
 
+def convert_string_to_int_array(char* str):
+    s = str.encode('utf-8').decode('utf-8')
+    intarr = np.empty([len(s), ], dtype=int)
+    for i, c in enumerate(s):
+        intarr[i] = ord(c)
+    return intarr
 
 cdef inline unsigned char str_1_get(unsigned char* s, Py_ssize_t i) nogil:
```

Comment:
nit: this seems to be unused now

Reply:
We will use it, because we are keeping the old APIs.